x-pack/test/apm_api_integration/tests/services/top_services.ts

```diff
@@ -6,30 +6,39 @@
  */
 
 import expect from '@kbn/expect';
-import { sortBy, pick, isEmpty } from 'lodash';
+import { sortBy } from 'lodash';
+import { service, timerange } from '@elastic/apm-synthtrace';
 import { APIReturnType } from '../../../../plugins/apm/public/services/rest/createCallApmApi';
 import { PromiseReturnType } from '../../../../plugins/observability/typings/common';
 import { FtrProviderContext } from '../../common/ftr_provider_context';
 import archives_metadata from '../../common/fixtures/es_archiver/archives_metadata';
 import { registry } from '../../common/registry';
+import { ENVIRONMENT_ALL } from '../../../../plugins/apm/common/environment_filter_values';
 
 export default function ApiTest({ getService }: FtrProviderContext) {
   const supertest = getService('legacySupertestAsApmReadUser');
+
+  const apmApiClient = getService('apmApiClient');
+  const synthtrace = getService('synthtraceEsClient');
+
   const supertestAsApmReadUserWithoutMlAccess = getService(
     'legacySupertestAsApmReadUserWithoutMlAccess'
   );
 
   const archiveName = 'apm_8.0.0';
 
-  const range = archives_metadata[archiveName];
+  const archiveRange = archives_metadata[archiveName];
 
   // url parameters
-  const start = encodeURIComponent(range.start);
-  const end = encodeURIComponent(range.end);
+  const archiveStart = encodeURIComponent(archiveRange.start);
+  const archiveEnd = encodeURIComponent(archiveRange.end);
+
+  const start = '2021-10-01T00:00:00.000Z';
+  const end = '2021-10-01T00:05:00.000Z';
```

Comment:
Feels a little strange to mix archives and runtime generated data. Isn't it a problem to have separate date ranges? 

I think this is fine for now but ideally I think we'd create an archive that includes ML data and synthtrace data